tslint.json

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": ["tslint:latest", "tslint-react"],
```

Comment:
# Comment
Awesome! This should work perfectly with the project :grin:

Reply:
I have to edit my previous statement haha, I just realized this is extending `tslint:latest` as opposed to `tslint-config-airbnb` like the other projects (and isn't therefore extending the airbnb guide like the other projects).